modules/core_project_factory/main.tf

```diff
@@ -108,10 +108,10 @@ module "project_services" {
 /******************************************
   Shared VPC configuration
  *****************************************/
-resource "time_sleep" "wait_5_seconds" {
+resource "time_sleep" "wait_vpc_sc_propagation" {
```

Comment:
@daniel-cit can we keep the same resource name so we can release this in a minor release?
We can add a comment to clean up for next breaking change.

Reply:
resource rename change reverted